test/functional/p2p_feefilter.py

```diff
@@ -9,12 +9,16 @@
 
 from test_framework.messages import MSG_TX, msg_feefilter
 from test_framework.mininode import mininode_lock, P2PInterface
-from test_framework.test_framework import BitcoinTestFramework
+from test_framework.test_framework import (
+    BitcoinTestFramework,
+    assert_equal,
+)
```

Comment:
This runs, but ISTM no other tests import this way and instead do:
```python
from test_framework.test_framework import BitcoinTestFramework
from test_framework.util import assert_equal
```

Reply:
Whoopsie. Fixed!